pycloudlib/instance.py

```diff
@@ -41,6 +41,12 @@ def __init__(self, key_pair, username: Optional[str] = None):
         self.connect_timeout = 60
         self.banner_timeout = 60
 
+    def __enter__(self):
+        return self
+
+    def __exit__(self, _type, _value, _traceback):
+        self.delete()
```

Comment:
This needs try/except: pass gating as there may be integration tests that already manually delete and instance before exiting the context mgr.

Reply:
flake will tell you you need docstrs on these